projectile.el

```diff
@@ -4398,13 +4454,20 @@ to run the replacement."
                         (file-name-as-directory
                          (read-directory-name "Replace in directory: "))
                       (projectile-acquire-root)))
+         (file-ext (if arg
```

Comment:
Perhaps instead of reusing the other arg you want to use double prefix arg or something like this here? At any rate you'll also have to update the command doc, otherwise that would seem weird.

Reply:
I think that using the double prefix arg is generally not natural: the user needs to pause to think about whether to use `C-u` or `C-u C-u`.

Instead, I revise the code to print a more suggestive message saying that if the users just need to enter an empty string to select all file extensions:

```
(file-ext (if arg
                       (if (fboundp #'helm-grep-get-file-extensions)
                           (car (helm-grep-get-file-extensions (list directory)))
                         (read-string
                          (projectile-prepend-project-name
                           "With file extension (empty string means all files): ")))
                     nil))
```

How do you think about that?